jfx_bridge/bridge.py

```diff
@@ -1157,7 +1186,21 @@ def __init__(self, bridge_conn, obj_dict):
         self._bridge_overrides = dict()
 
     def __getattribute__(self, attr):
-        if attr.startswith(BRIDGE_PREFIX) or attr == "__class__" or attr in BridgedObject._DONT_BRIDGE or attr in BridgedObject._LOCAL_METHODS or (attr in BridgedObject._DONT_BRIDGE_UNLESS_IN_ATTRS and attr not in self._bridge_attrs):
+
+        if attr.startswith(BRIDGE_PREFIX):
+            return object.__getattribute__(self, attr)
+
+        try:
+            # Try calling a registered override
+            return self._bridge_conn.external_overrides[attr](self)
+        except ConcedeOverride:
+            # The override signalled that it does not want to override this case
+            pass
+        except KeyError:
+            # There was no override for this attribute
+            pass
```

Comment:
```suggestion
        if attr in self._bridge_conn.external_overrides:
            try:
                # Try calling a registered override
                return self._bridge_conn.external_overrides[attr](self)
            except ConcedeOverride:
                # The override signalled that it does not want to override this case
                pass
```
This way, KeyError exceptions from buggy overrides won't be hidden - the only "exception" that should be allowed is ConcedeOverride

Reply:
Oh, good call, didn't realize that this changed the behavior if an override wants to return KeyError. Thanks!